tests/components/backup/test_websocket.py

```diff
@@ -313,8 +314,11 @@ async def test_agents_download(
         }
     )
     with patch.object(BackupSyncAgentTest, "async_download_backup") as download_mock:
-        assert await client.receive_json()
-        assert snapshot == download_mock.call_args
+        assert snapshot == await client.receive_json()
```

Comment:
Please put the expected value, in this case the snapshot, on the right side of the assertion.